### PR TITLE
[requirements] Update dependencies & drop python 2 support

### PR DESCRIPTION
**Pull request by Weirdo914**
Dropped python2 support and updated requirements.
Fixes https://github.com/openwisp/openwisp-utils/issues/43

### CODE REVIEW

File: tests/test_project/tests/test_admin.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 from django.test import TestCase
 from django.urls import reverse
 
-from . import CreateMixin
 from ..models import Operator, Project, RadiusAccounting
+from . import CreateMixin
```
Comment (dwang) [inline tests/test_project/tests/test_admin.py]: Is there any particular reason for this change?

File: tests/test_project/tests/test_api.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
 from openwisp_users.tests.utils import TestOrganizationMixin
 from test_project.serializers import ShelfSerializer
 
-from . import CreateMixin
 from ..models import Shelf
+from . import CreateMixin
```
Comment (dwang) [inline tests/test_project/tests/test_api.py]: Is there any particular reason for this change?

Comment (Weirdo914) [reply tests/test_project/tests/test_api.py]: There was an error in build-
`ERROR: /home/travis/build/Weirdo914/openwisp-utils/tests/test_project/tests/test_api.py Imports are incorrectly sorted.`

File: tests/test_project/tests/test_admin.py (hunk 1 above)
Comment (Weirdo914) [reply tests/test_project/tests/test_admin.py]: There was an error in build-
`ERROR: /home/travis/build/Weirdo914/openwisp-utils/tests/test_project/tests/test_admin.py Imports are incorrectly sorted.`

**Comment by atb00ker** [comment]
@Weirdo914 I made a mistake in review here.
Please check and remove `from __future__ import unicode_literals` as well! :smile:

File: setup.py [hunk 3]
```diff
@@ -41,7 +41,7 @@
     zip_safe=False,
     install_requires=['django-model-utils>=3.1.2,<3.3.0'],
```
Comment (nemesisdesign) [inline setup.py]: update this as well: `django-model-utils>=3.1.2,<4.1.0`

File: .travis.yml [hunk 4]
```diff
@@ -4,20 +4,12 @@ cache: pip
 
 python:
   - "3.5"
```
Comment (nemesisdesign) [inline .travis.yml]: let's use python 3.7

**Comment by Weirdo914** [comment]
Sure looks like the travis is failing due to some other errors do you want me to upgrade this also to django3 with the other reqirements in django-x509 because i think the same errors will arise in this one. @nemesisdesign

File: .travis.yml [hunk 5]
```diff
@@ -2,20 +2,13 @@ language: python
...
   - DJANGO="django>=2.0,<2.1"
   - DJANGO="django>=2.1,<2.2"
-
-matrix:
-  exclude:
-    - python: "2.7"
-      env: DJANGO="django>=2.0,<2.1"
-    - python: "2.7"
-      env: DJANGO="django>=2.1,<2.2"
+  - DJANGO="django>=3.0"
```
Comment (nemesisdesign) [inline .travis.yml]: this should be `- DJANGO="django>=3.0,<3.1"

Comment (Weirdo914) [reply .travis.yml]: @nemesisdesign Ok I looked at django-allauth seems like their latest version doesnt support djanngo 3 and it is installed here `pip install https://github.com/openwisp/openwisp-users/tarball/master`

Comment (nemesisdesign) [reply .travis.yml]: Try with the development version @waleko

**Comment by Weirdo914** [comment]
Now its failing because of openwisp-users

**Comment by oltarzewskik** [comment]
@Weirdo914 It looks like you can delete this line: https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/staticfiles.py#L28 entirely to avoid errors.

look at this for more details:
https://github.com/openwisp/openwisp-utils/pull/64/files#r361649867

And BTW the line was written badly from begging because it should look like this:
`super(DependencyFinder, self).__init__(*args, **kwargs)` instead of:
`super(FileSystemFinder, self).__init__(*args, **kwargs)`

**Comment by atb00ker** [comment]
@Weirdo914 
```
'Programming Language :: Python :: 3.7',
```
should be
```
'Programming Language :: Python :: 3',
```

**Comment by Weirdo914** [comment]
I dont understand why the coverage is decreasing

**Comment by atb00ker** [comment]
> I dont understand why the coverage is decreasing

That much difference is insignificant, you can ignore it but please rebase with master.

**Comment by Weirdo914** [comment]
I rebased with master

**Pull request merged**